hooks/running.yml

```diff
@@ -11,10 +11,8 @@
     version: '{{ ansible_commit }}'
 
 - name: Run the integration test suite
-  shell: 'source ansible/hacking/env-setup && ansible-playbook -i inventory run_{{ platform }}_integration_tests.yaml'
+  shell: 'ansible/test/runner/ansible-test network-integration --continue-on-error --inventory /var/lib/nodepool/ansible-testrunner/inventory {{ platform }}_.* -v'
```

Comment:
You should also be able to use `network/{{ platform }}/` as the target, which might be easier to read compared to using the regex match.